Tests/CommandLineTests/ConvertActionTests.swift

```diff
@@ -3280,3 +3434,15 @@ private extension ConvertAction {
         return try await perform(logHandle: &logHandle)
     }
 }
+
+extension RenderIndex {
```

Comment:
minor: If the test was added somewhere inside of SwiftDocCTests instead, then we wouldn't need to add another copy of these.